packages/rehype-mathjax/package.json

```diff
@@ -34,19 +34,22 @@
   ],
   "files": [
     "lib/",
+    "*.d.ts",
     "browser.js",
     "chtml.js",
     "svg.js",
     "index.js"
   ],
   "main": "index.js",
+  "types": "index.d.ts",
   "dependencies": {
+    "@types/mathjax": "^0.0.36",
     "hast-util-from-dom": "^2.0.0",
     "hast-util-to-text": "^2.0.0",
     "jsdom": "^16.0.0",
     "mathjax-full": "^3.0.0",
+    "unified": "^9.0.0",
```

Comment:
I don’t think we should put `unified` in dependencies, I suggest to put something in the docs that folks install it themselves.